plugins/modules/compute_resource.py

```diff
@@ -173,6 +173,7 @@
       - ACME
     provider: vmware
     provider_params:
+      caching_enabled: 0
```

Comment:
If it's `bool`, it's probably better to have the example say `caching_enabled: False`

Reply:
You´re right. Applied